crates/tako/src/messages/worker.rs

```diff
@@ -63,12 +65,12 @@ pub struct TaskIdMsg {
 }
 
 #[derive(Serialize, Deserialize, Debug)]
-//#[serde(tag = "op")]
 pub enum ToWorkerMessage {
     ComputeTask(ComputeTaskMsg),
     StealTasks(TaskIdsMsg),
     CancelTasks(TaskIdsMsg),
     NewWorker(NewWorkerMsg),
+    Reservation(bool),
```

Comment:
What does this bool mean? Maybe use an enum?

Reply:
It is just on/off flag.

Reply:
It's not clear from the type alone what should this do. It would be nice to add a separate type or at least a comment. Also the other worker message are verbs (Cancel, Compute, Steal), but this one is not (it should also be fixed for `NewWorker`.

Maybe at least `SetReservation`?

Reply:
``SetReservation`` sounds good